.gitlab-ci.yml

```diff
@@ -3,6 +3,9 @@ stages:
   - binary_build
   - integration_test
   - package_build
+  - deploy_testing
```

Comment:
These step names are pretty non-explicit, could we name them `testkitchen-{deploy|test|cleanup}` instead?

Reply:
Yes we can